riff system install segfaults

The `riff system install` and `uninstall` commands are segfaulting after #788

```
$ riff system install
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1c36e20]

goroutine 1 [running]:
github.com/projectriff/riff/cmd/commands.SystemInstall.func1(0xc000316280, 0x2b255e0, 0x0, 0x0, 0x0, 0x0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/cmd/commands/system.go:56 +0x30
github.com/projectriff/riff/cmd/commands.CreateAndWireRootCommand.func2.1(0xc000316280, 0x2b255e0, 0x0, 0x0, 0x0, 0x0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/cmd/commands/wiring.go:180 +0x58
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).execute(0xc000316280, 0x2b255e0, 0x0, 0x0, 0xc000316280, 0x2b255e0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/projectriff/riff/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc00047f180, 0x2, 0xc0004203c0, 0xc0004203a0)
        /Users/scothis/Development/go/src/github.com/projectriff/riff/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
main.main()
        /Users/scothis/Development/go/src/github.com/projectriff/riff/main.go:31 +0x2b
```

--- a/cmd/commands/wiring.go
+++ b/cmd/commands/wiring.go
@@ -145,6 +145,7 @@ See https://projectriff.io and https://github.com/knative/docs`,
 	)
 
 	system := System()
+	installKubeConfigSupport(system, &client, &kc)
 	system.AddCommand(
 		SystemInstall(&kc),
 		SystemUninstall(&kc),

--- a/docs/riff_system.md
+++ b/docs/riff_system.md
@@ -9,7 +9,9 @@ Manage system related resources
 ### Options
 
 ```
-  -h, --help   help for system
+  -h, --help              help for system
+      --kubeconfig path   the path of a kubeconfig (default "~/.kube/config")
+      --master address    the address of the Kubernetes API server; overrides any value in kubeconfig
 ```
 
 ### SEE ALSO

--- a/docs/riff_system_install.md
+++ b/docs/riff_system_install.md
@@ -35,6 +35,13 @@ riff system install [flags]
       --node-port         whether to use NodePort instead of LoadBalancer for ingress gateways
 ```
 
+### Options inherited from parent commands
+
+```
+      --kubeconfig path   the path of a kubeconfig (default "~/.kube/config")
+      --master address    the address of the Kubernetes API server; overrides any value in kubeconfig
+```
+
 ### SEE ALSO
 
 * [riff system](riff_system.md)	 - Manage system related resources

--- a/docs/riff_system_uninstall.md
+++ b/docs/riff_system_uninstall.md
@@ -26,6 +26,13 @@ riff system uninstall [flags]
       --istio   include Istio and the istio-system namespace in the removal
 ```
 
+### Options inherited from parent commands
+
+```
+      --kubeconfig path   the path of a kubeconfig (default "~/.kube/config")
+      --master address    the address of the Kubernetes API server; overrides any value in kubeconfig
+```
+
 ### SEE ALSO
 
 * [riff system](riff_system.md)	 - Manage system related resources

